### ISSUE TITLE
Can't use grenades

### ISSUE THREAD

**Issue by jazluvr449** [bfeber/HLA-NoVR#135]
Since the implementation of wristpockets to store items, I can store 2 'nades, but i can't use, pick up or throw/arm them. Well, i can only throw one that i have in my wrist, but then i gotta pick up another one, to store in wrist to use. Very tedious and unforgiving in a firefight. Can't use the grenade launcher with them either.

Was much better the old way, but still being able to store a couple would be great, if do-able. But this is sure death in a firefight.
Edit------
I take some of that back. I'm getting used to the new way. The grenade launcher is a problem still.

**Comment by jazluvr449** [comment]
I'm at Arms Race, and I beat the 1st few enemies, but when I proceed to the outdoor area, it looks like I've enabled noclip, which i never use anymore. I float around, I see where i need to go, (i think), but I Can't make any further progress. Also I previously was able to arm the shotgun with 1 grenade, and have 2 in wrists, but that just....disappeared on in next chapter. 

You guys are doing an awesome job! I can't stop playing! I want to wait until it's all finished, but I can't stay away from it...lol.
